mobile/src/bundles/common/styles/fonts.ts

```diff
@@ -67,6 +67,11 @@ const FONTS = {
         color: Color.PRIMARY,
         textTransform: 'uppercase' as const,
     },
+    [TextCategory.INPUT_PLACEHOLDER]: {
```

Comment:
```suggestion
    [TextCategory.PLACEHOLDER]: {
```
We can have placeholders not only in imports